usagov/vote-gov-drupal
# VOTE-3141 hide NVRF svg icons from screen readers

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-3141

## Description

Hide NVRF svg icons from screen readers

## Deployment and testing

### Post-deploy steps

1. `lando drush cr`

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/register/minnesota/mail-in-form-filler and confirm that the icon list svgs have `aria-hidden` on them.

## Checklist for the Developer




## Checklist for the Peer Reviewers
